src/renderers/webgl-fallback/nodes/GLSLNodeBuilder.js

```diff
@@ -476,7 +476,7 @@ ${ flowData.code }
 
 			const array = dataAttribute.array;
 
-			if ( ( array instanceof Uint32Array || array instanceof Int32Array || array instanceof Uint16Array || array instanceof Int16Array ) === false ) {
+			if ( ( array instanceof Uint32Array || array instanceof Int32Array || array instanceof Int16Array ) === false ) {
```

Comment:
This just breaks `Uint16Array` support in the WebGL Backend. And now generate this kind of error:
```
[.WebGL-0x13000c4ea00] GL_INVALID_OPERATION: Vertex shader input type does not match the type of the bound vertex attribute.
```

 Are you sure about this one? /cc @Mugen87

Reply:
This change was required to make compressed models work with the WebGL backend so I would say the previous approach was wrong.

Do you mind demonstrating with a fiddle how the error occurs?

In any event, the GLSL builder should only generate `iuvec*` if the `gpuType` is `IntType`  or when `Uint32Array` and `Int32Array` is used. In all other cases, the shader type should be float.